app/services/bulk_upload/lettings/validator.rb

```diff
@@ -101,8 +103,8 @@ def validate_duplicate_rows
     end
   end
 
-  def any_logs_invalid?
-    row_parsers.any? { |row_parser| row_parser.log.invalid? }
+  def invalid_row_numbers
+    row_parsers.each_with_index.map { |row_parser, index| index + row_offset + 1 if row_parser.log.invalid? }.compact
```

Comment:
Only thought is that `any?` will stop as soon as it finds a match, so might be a bit more performant, but given this very rarely returns true, it's probably okay to change it to help with the logging